packages/web-app-files/src/components/Spaces/QuotaModal.vue

```diff
@@ -68,87 +47,35 @@ export default defineComponent({
   },
   data: function () {
     return {
-      selectedOption: {},
-      options: []
+      selectedOption: 0
     }
   },
   computed: {
     confirmButtonDisabled() {
-      return this.space.spaceQuota.total === this.selectedOption.value
+      return this.space.spaceQuota.total === this.selectedOption
     },
     modalTitle() {
       return this.$gettextInterpolate(this.$gettext('Change quota for space %{name}'), {
         name: this.space.name
       })
-    },
-    DEFAULT_OPTIONS() {
-      return [
-        {
-          displayValue: '1',
-          displayUnit: 'GB',
-          value: Math.pow(10, 9)
-        },
-        {
-          displayValue: '5',
-          displayUnit: 'GB',
-          value: 5 * Math.pow(10, 9)
-        },
-        {
-          displayValue: '10',
-          displayUnit: 'GB',
-          value: 10 * Math.pow(10, 9)
-        },
-        {
-          displayValue: '50',
-          displayUnit: 'GB',
-          value: 50 * Math.pow(10, 9)
-        },
-        {
-          displayValue: '100',
-          displayUnit: 'GB',
-          value: 100 * Math.pow(10, 9)
-        },
-        {
-          displayValue: '500',
-          displayUnit: 'GB',
-          value: 500 * Math.pow(10, 9)
-        },
-        {
-          displayValue: '1000',
-          displayUnit: 'GB',
-          value: 1000 * Math.pow(10, 9)
-        },
-        {
-          displayValue: this.$gettext('No restriction'),
-          displayUnit: '',
-          value: 0,
-          unlimited: true
-        }
-      ]
     }
   },
   mounted() {
-    this.setOptions()
+    this.selectedOption = this.space.spaceQuota.total
```

Comment:
```suggestion
    this.selectedOption = this.space.spaceQuota.total || 0
```
We want an int here, right? So if `total` is `undefined` we need to fall back to `0` then.